services/search-builder.js

```diff
@@ -43,14 +51,17 @@ function SearchBuilder(model, opts, params, searchFields) {
                 query[subField.field] = new RegExp('.*' + params.search + '.*',
                   'i');
                 elemMatch.$elemMatch.$or.push(query);
```

Comment:
I'm not sure how easy it is to apply the same refacto logic as the sequelize liana because of the `elemMatch` and `orQuery`